src/main/java/net/glowstone/util/lang/LanguageProvider.java

```diff
@@ -12,7 +13,7 @@
     /**
      * Returns the internationalized message according to the effective locale of the player.
      */
-    String getString(String key, GlowPlayer p, Object ... args);
+    String getString(GlowPlayer p, String key, Object ... args);
```

Comment:
@momothereal Thank you. I amended the javadoc to add @param's and @return. I also renamed p to player. At the moment I am using the code which was in the previous pull request. I haven't modified the backend engine much.

What do you think about creating a static i18n `I` class? I've seen this done by other i18n implementations. This would reduce code clutter, from `GlowServer.lang.getString` to `I.tr`.